intellst-api/src/Service/Register.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Service;
+
+use App\Entity\User;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+use Doctrine\ORM\getDoctrine;
+
+class Register extends AbstractController
+{
+    public function __construct(RequestStack $requestStack, UserPasswordEncoderInterface $encoder)
+    {
+        $this->requestStack = $requestStack;
+        $this->encoder = $encoder;
+    }
+
+    public function register() : string
```

Comment:
1. spacing
2. please test. it will fail immediately as expected type "string" != returned type "Response"